File: app/src/main/java/com/gianessi/stargazers/models/Repo.java
package com.gianessi.stargazers.models;

import com.google.gson.annotations.SerializedName;

public class Repo {

    public static final String NAME = "reponame";

    @SerializedName("id")
    private long id;
    @SerializedName("name")
    private String name;

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/gianessi/stargazers/models/Repo.kt
package com.gianessi.stargazers.models

import com.google.gson.annotations.SerializedName



data class Repo(
        @SerializedName("id") val id: Long,
        @SerializedName("name") val name: String?) {

    companion object {
        const val NAME = "reponame"
    }
}
